openmc/geometry.py

```diff
@@ -257,7 +257,49 @@ def get_all_lattices(self):
                     lattices[cell.fill.id] = cell.fill
 
         return lattices
+    
+    def get_all_surfaces(self):
+        """
+        Return all surfaces used in the geometry
+
+        Returns
+        -------
+        collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
 
+        """
+        surfaces = OrderedDict()
+        
+        for cell in self.get_all_cells().values():
+            self._get_surfaces_from_region(surfaces, cell.region)
+        return surfaces
+    
+    def _get_surfaces_from_region(self, surfaces, region):
```

Comment:
This doesn't need to be a method of `Geometry` itself since it doesn't actually use any information from the geometry class. You can either make it a module-level private function, or, since it is only used in `get_all_surfaces`, you can actually just define it within the body of that method, i.e.
```Python
def get_all_surfaces(...):
    ...
    def _get_surfaces_from_region(surfaces, region):
        ...

    surfaces = OrderedDict()
    ...
```